site/en/r2/tutorials/keras/overfit_and_underfit.ipynb

```diff
@@ -222,225 +267,291 @@
         "\n",
         "To find an appropriate model size, it's best to start with relatively few layers and parameters, then begin increasing the size of the layers or adding new layers until you see diminishing returns on the validation loss. Let's try this on our movie review classification network. \n",
         "\n",
-        "We'll create a simple model using only ```Dense``` layers as a baseline, then create smaller and larger versions, and compare them."
+        "We'll create a simple model using only ```Dense``` layers as a baseline, then create smaller and larger versions, and compare them. ** * Tip: If a L<sup>th</sup> Dense layer in your network has n number of hidden units, then any layer before this layer shouldn't have less than n hidden units as it will become the bottleneck.***"
       ]
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "_ReKHdC2EgVu"
       },
+      "cell_type": "markdown",
       "source": [
         "### Create a baseline model"
       ]
     },
     {
-      "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
-        "colab": {},
         "colab_type": "code",
-        "id": "QKgdXPx9usBa"
+        "id": "QKgdXPx9usBa",
+        "colab": {}
       },
-      "outputs": [],
+      "cell_type": "code",
       "source": [
-        "baseline_model = keras.Sequential([\n",
-        "    # `input_shape` is only required here so that `.summary` works. \n",
-        "    keras.layers.Dense(16, activation=tf.nn.relu, input_shape=(NUM_WORDS,)),\n",
-        "    keras.layers.Dense(16, activation=tf.nn.relu),\n",
-        "    keras.layers.Dense(1, activation=tf.nn.sigmoid)\n",
-        "])\n",
-        "\n",
-        "baseline_model.compile(optimizer='adam',\n",
+        "def create_baseline_model():\n",
+        "  baseline_model = Sequential()\n",
```

Comment:
Hey,

.add() just adds more code when its not necessary. I think we should stick to the original version of creating the models here and everywhere below.

Reply:
Hey,

Thanks for reviewing it. It certainly adds few words to the code but then there is a difference between `ugly` code and `good` code. After using `Keras` for a long time, I think this is the actual way all the people(especially everyone I know on `Kaggle`) use it. Regarding the `dropout`, I think I would remove it and restate it in the original state. Showing the internals wasn't necessary for the notebook, I certainly overthought it.